src/dcos_e2e/docker_utils/__init__.py

```diff
@@ -0,0 +1,122 @@
+from typing import Any, Dict, Optional, Tuple
+
+import docker
+
+
+class DockerLoopbackVolume():
+    """
+    A loopback device sidecar, created in a Docker container.
+    """
+
+    def __init__(
+        self,
+        size: int,
+        labels: Optional[Dict[str, str]] = None,
+    ) -> None:
+        """
+        Create a loopback device pointing to a block device running
+        in a container. This can be used to provide an unformatted
+        disk to a cluster.
+
+        Args:
+            size: Size of the block device in Megabytes.
+            labels: Docker labels to add to the container.
+        """
+        self.size = size
+        docker_container_labels = labels or {}
+
+        self.container, self.path = \
+            DockerLoopbackVolume.create(size, docker_container_labels)
+
+    @staticmethod
```

Comment:
Why use `staticmethod` for`create` and `destroy`?

Reply:
They're needed for the CLI. There we can't  use an instance of `DockerLoopbackVolume` when destroying an existing sidecar, because it would create a new sidecar during instantiation.
An alternative would be to have multiple constructors (or a factory) for `DockerLoopbackVolume`, allowing us to create an instance from an existing `docker.models.containers.Container`.